src/instructlab/cli/model/chat.py

```diff
@@ -101,6 +102,45 @@
     "--temperature",
     cls=clickext.ConfigOption,
 )
+@click.option(
+    "--rag",
+    "rag_enabled",
+    default=False,
```

Comment:
unsure if you need a default for `is_flag` types, I think the default is assumed to be false

Reply:
I honestly think this is a case where it's good to keep it, much more explicit for folks working with the code